include/atomics/receiver_cadmium.hpp

```diff
@@ -25,48 +35,77 @@
 
 #include "../data_structures/message.hpp"
 
+/** 
+* namespace is used to organize code into logical groups and to prevent name collisions.
+*/
 using namespace cadmium;
 using namespace std;
 
-//Port definition
+/**
+* structure consists of class public output and input Messages
+* Port definition 
+*/
     struct receiver_defs {
 		struct out : public out_port<message_t> {
 		};
 		struct in : public in_port<message_t> {
 		};
 	};
    
+
     template<typename TIME>
+    
     class Receiver {
-        using defs = receiver_defs; // putting definitions in context
+
+        /** 
+        * putting reciever definitions in context. 
+        */
+        using defs = receiver_defs;  
```

Comment:
Place comment after member like:
using defs = receiver_defs; /**< putting reciever definitions in context. */